src/platform/cannonlake/include/platform/memory.h

```diff
@@ -203,10 +203,12 @@
 #define SRAM_SW_REG_SIZE	0x1000
 
 /* SRAM window 0 FW "registers" */
-#define SRAM_REG_ROM_STATUS	0x0
-#define SRAM_REG_FW_STATUS	0x4
-#define SRAM_REG_FW_TRACEP	0x8
-#define SRAM_REG_FW_END		0xc
+#define SRAM_REG_ROM_STATUS		0x0
+#define SRAM_REG_FW_STATUS		0x4
+#define SRAM_REG_FW_TRACEP		0x8
+#define SRAM_REG_FW_IPC_RECEIVED	0xc
```

Comment:
Maybe it'd be better to have `COUNT`/`NUM` suffix here. I know it may seem verbose but at least no1 will think that's some kind of boolean or ipc payload.

Reply:
@jajanusz 
I've change ```SRAM_REG_FW_IPC_RECEIVED/PROCESSED``` to ```SRAM_REG_FW_IPC_RECEIVED/PROCESSED_COUNT```.

Reply:
Thanks